languages/js/src/helpers.test.ts

```diff
@@ -0,0 +1,29 @@
+import { ancestors } from './helpers';
```

Comment:
Not really related to this PR; just noticed we didn't have any tests for `ancestors()` as I was fiddling with the types on it